.github/workflows/_vllm-benchmark.yml

```diff
@@ -0,0 +1,233 @@
+name: vllm-benchmark
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        required: true
+        type: string
+        description: The runner to use.
+      docker_image:
+        required: true
+        type: string
+        description: The name of the base docker image to use.
+      build_environment:
+        required: true
+        type: string
+        description: The build environment name, e.g. linux-jammy-cuda12.9-py3.12-gcc11
+      pytorch_branch:
+        required: false
+        type: string
+        description: The PyTorch branch to checkout. Default to the current checkout.
+      pytorch_commit:
+        required: false
+        type: string
+        description: The PyTorch commit to checkout. Default to the current checkout.
+      models:
+        required: false
+        type: string
+        description: The list of models to benchmark. Default to all models.
+        default: ''
+
+jobs:
+  benchmark:
+    runs-on: ${{ inputs.runner }}
+    permissions:
+      id-token: write
+      contents: read
+    container:
+      image: ${{ inputs.docker_image }}
+      options: --gpus all --ipc=host --tty
```

Comment:
I'm curious if these are always necessary to set

Reply:
What I have observed about these options are:

* `--gpus all` is always needed if there are GPUs
* `--ipc=host`, optional
* `--tty`, optional.  I originally thought that I needed this to have the logs continuously streaming to GitHub, but I don't think that's the case